crates/ty/docs/configuration.md

```diff
@@ -153,6 +153,64 @@ typeshed = "/path/to/custom/typeshed"
 
 ## `src`
 
+#### `exclude`
+
+A list of file and directory patterns to exclude from type checking.
+
+Patterns follow a syntax similar to `.gitignore`:
+- `./src/` matches only a directory
+- `./src` matches both files and directories
+- `src` matches files or directories named `src` anywhere in the tree (e.g. `./src` or `./tests/src`)
+- `*` matches any number of characters except `/`
+- `**` matches any number of characters including `/`
+- `?` matches a single character except `/`
+- `[abc]` matches any character in the set
+- `!pattern` negates a pattern (includes files that would otherwise be excluded)
```

Comment:
What about `{a,b,...}`?

Reply:
Oh nevermind, you're using your own portable glob syntax that doesn't have this. Makes sense.